db/docs/source/introduction.rst

```diff
@@ -62,6 +62,26 @@ The source imagery is linked via the ``capture_source_id`` attribute of building
 The NZ Aerial Surveys data dictionary is here: https://nz-imagery-surveys.readthedocs.io/en/latest/index.html
 
 
+Locality
+--------
+Locality includes suburb, locality, island and park reserve. Water features such as coastal bay, lake and inland bay are not included.
+
```

Comment:
Consider including some additional background on locality.
For example: "Building outlines include the attribute `suburb_locality` which provides the suburb or locality name as sourced from NZ Localities (an NZ Fire Service owned dataset). The attribute may refer to suburbs, localities, islands and park reserves. Water features..."